content/build/cloud/usage.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Building with Cloud Build
+description: Invoke your cloud builds with the Buildx CLI client
+keywords: build, cloud build, usage, cli, buildx, client
+aliases:
+  - /hydrobuild/usage/
+---
+
+To build using Docker Cloud Build, invoke a build command and specify the name of the
+builder using the `--builder` flag.
+
+```console
+$ docker buildx build --builder cloud-<ORG>-default --tag <IMAGE> .
+```
+
+## Use by default
+
+If you want to use Docker Cloud Build without having to specify the `--builder` flag
+each time, you can set it as the default builder.
+
+{{< tabs group="ui" >}}
+{{< tab name="CLI" >}}
+
+Run the following command:
+
+```console
+$ docker buildx use cloud-<ORG>-default --global
+```
+
+{{< /tab >}}
+{{< tab name="Docker Desktop" >}}
+
+1. Open the Docker Desktop settings and navigate to the **Builders** tab.
+2. Find the cloud builder under **Available builders**.
+3. Open the drop-down menu and select **Use**.
+
+   ![Selecting the cloud builder as default using the Docker Desktop GUI](/build/images/set-default-builder-gui.webp)
+
+{{< /tab >}}
+{{< /tabs >}}
+
+Changing your default builder with `docker buildx use` only changes the default
+builder for the `docker buildx build` command. The `docker build` command still
+uses the `default` builder, unless you specify the `--builder` flag explicitly.
+
+If you use build scripts, such as `make`, we recommend that you update your
+build commands from `docker build` to `docker buildx build`, to avoid any
+confusion with regards to builder selection. Alternatively, you can run `docker
+buildx install` to make the default `docker build` command behave like `docker
+buildx build`, without discrepancies.
+
+## Loading build results
+
+Building with `--tag` loads the build result to the local image store
+automatically when the build finishes. To build without a tag and load the
+result, you must pass the `--load` flag.
```

Comment:
This is done by default when using the cloud driver, you don't have to specify the `--load` flag.

Reply:
Resolved 👍